src/20231113_Multi_IncreaseStablecoinOptimalBorrowRates/IncreaseStablecoinOptimalBorrowRates.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "Increase Stablecoin Optimal Borrow Rates"
+author: "monet-supply + Gauntlet (impl: Alice Rozengarden (@Rozengarden - Aave-chan initiative))"
+discussions: "https://governance.aave.com/t/arfc-increase-optimal-borrow-rates-for-ethereum-stablecoin-markets/15096/3"
+---
+
+## Simple Summary
+
+This AIP propose to modifie the parameters of stablecoins accross all the aave markets. Mainly by setting the slope1 to 5%, as well as changing the RF and Uopt of some of them.
```

Comment:
I would specify more on the RF change and uOpt (maybe as its explained on snapshot: https://snapshot.org/#/aave.eth/proposal/0xb9b28f57f7633dd6b987de9abcede23da62fe4fab6b002f189b8b25a7c02ea93 )

Reply:
I tried, let me know If it's enought